app/routes/admin/delete/owners.js

```diff
@@ -1,17 +1,13 @@
 const { routes, views } = require('../../../constants/admin')
 const { admin } = require('../../../auth/permissions')
-// const { getUser } = require('../../../auth')
```

Comment:
Can these commented lines be removed, or are you waiting for extra functionality in later stories?

Reply:
check the right hand column 😄